16-draft.json

```diff
@@ -370,6 +370,13 @@
           "examples": [
             "https://t.me/example-channel"
           ]
+        },
+        "instagram": {
+          "description": "URL to Instagram profile",
+          "type": "string",
+          "examples": [
+            "https://www.instagram.com/examplespace"
+          ]
         }
```

Comment:
While I'm not a fan of Instagram at all (since the Meta takeover), I can see that some spaces might have an account. As such, it probably makes sense to add it to the schema.

Reply:
At the moment, a total of 7 spaces in the directory list an Instagram account as `contact.instagram` or `contact.ext_instagram`.  A slight majority seems to go by URL rather than account name only:

```
"ext_instagram": "leighhackspace"
"instagram": "https://www.instagram.com/netz_39/"
"instagram": "https://www.instagram.com/farsetlabs"
"instagram": "https://www.instagram.com/hackspace_siegen/"
"instagram": "segvaultspace"
"instagram": "https://www.instagram.com/hack.manhattan/"
"instagram": "@MakeSpaceMadrid"
```

I'm in favor of going with the URL over username only.